### DISCUSSION TITLE
Data quality allows double calibration

### DISCUSSION THREAD

**Discussion by Navid200** [NightscoutFoundation/xDrip#3026] [Q&A]
What could cause someone using a G6 in native mode get these logs?

![367694818_1502642157214697_5092020440641595753_n](https://github.com/NightscoutFoundation/xDrip/assets/51497406/cf81651d-cf2b-4ef9-bcfa-0f2cecf8c7c0)

![367383769_1502567990555447_8030737919371150906_n](https://github.com/NightscoutFoundation/xDrip/assets/51497406/ce323bc6-27a2-4d49-9506-c1217bebfb27)

**Comment by jamorham** [comment]
I think fallback to xDrip algorithm which I believe you have raised a PR to discontinue as it could easily cause problems for people.
